### PR TITLE
add known issue to 53 RNs

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-556

Description of Changes:
* Add the known issue for 5.3 release notes

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes53.md [hunk 1]
```diff
@@ -224,19 +224,24 @@ This section describes issues that users of {{site.productname}} 5.3 may encount
 
 **Outline**
 
-* [](#)
+* [Find and replace in selection feature not working as expected for inline editors and IE 11](#findandreplaceinselectionfeaturenotworkingasexpectedforinlineeditorsandie11)
 
 
-### 
+### Find and replace in selection feature not working as expected for inline editors and IE 11
```
Comment (lnewson) [inline release-notes/release-notes53.md:230]: I'm not sure I'd say "not working as expected", as it is working, there's just no visual representation due to how browsers natively work. That being said, I can't think of a better term atm :shrug:

Comment (metricjs) [reply release-notes/release-notes53.md:230]: No idea if it's better but maybe "Find and replace in selection feature missing content selection for inline editors and IE 11"?

Comment (lnewson) [reply release-notes/release-notes53.md:230]: Yeah, that sounds better

Comment (tylerkelly13) [reply release-notes/release-notes53.md]: Fixed in https://github.com/tinymce/tinymce-docs/pull/1537/commits/877c4459eeb8713c8e652aa83e444bfdcc7ae6f6

**Pull request merged**